resources/js/Components/Utilities/Datatable/Datatable.vue

```diff
@@ -136,6 +136,12 @@
 				default() {
 					return false
 				}
+			},
+			exportButton: {
+				type: Boolean,
```

Comment:
default for boolean doesnt need be a function

it can just be : 

`default: true`